calibration/CTDMOG/CGINS-CTDMOG-13422__20161015.csv

```diff
@@ -11,15 +11,15 @@ serial,name,value,notes
 37-13422,CC_j,2.803717e-005,
 37-13422,CC_p_range,1450,
 37-13422,CC_pa0,3.521130e-001,
-37-13422,CC_pa1,4.579698e-003,
+37-13422,CC_pa1,4.579699e-03,
 37-13422,CC_pa2,-2.295324e-011,
 37-13422,CC_ptca0,5.241072e+005,
-37-13422,CC_ptca1,6.134143e+000,
+37-13422,CC_ptca1,6.134144e+00,
 37-13422,CC_ptca2,-1.210638e-001,
 37-13422,CC_ptcb0,2.498438e+001,
 37-13422,CC_ptcb1,-1.250000e-004,
 37-13422,CC_ptcb2,0.00E+00,
 37-13422,CC_ptempa0,1.462932e+002,
 37-13422,CC_ptempa1,-7.408421e-002,
 37-13422,CC_ptempa2,2.248582e-006,
-37-13422,CC_wbotc,3.0815e-008,
+37-13422,CC_wbotc,3.081458e-08,
```

Comment:
Looks good.

Reply:
Looks good (CD and JL) verified with QCT